Jsonparser.cpp

```diff
@@ -0,0 +1,194 @@
+#include "Jsonparser.h"
+
+Jsonparser::Jsonparser(std::ifstream & f) {
+	int keyv, i;
+	std::string::size_type found;
+	if (!f.good()) throw 56;
+	else {
+		std::string sor = "";
+		std::string tmp;
+
+		while (!f.eof()) {
+			getline(f, sor);
+			keyv = 0;
+			found = sor.find('"');
+			if (found != std::string::npos) {
+				i = found + 1;
+				tmp = "";
+				while (sor[i] != '"') {
+					tmp = tmp + sor[i];
+					i++;
+				}
```

Comment:
Nem biztos, hogy lesz abban stringben egy második idézőjel is.

Reply:
@wajzy a feladatkiírás szerint a kulcs két idézőjel között lévő szöveg, és az érték keresésnél lekezeli azt is ha van idézőjel,azt is ha nincsen. Nem sikerült értelmeznem sajnos, hogy ezzel mi a gond.

Reply:
Ha lekezeli, akkor jó. Hirtelen ránézve nem látom, mi fog történni, ha a sor így néz ki:
"name" : "error,
Tegyük fel, hogy az értéknél járunk, és found tárolja a 3. " helyét. Ezután bemásoljuk tmp-be az error betűit, majd mivel nincs több idézőjel, túlindexeljük a sor tömböt.